### PR TITLE
add webhook model and job

### PR DESCRIPTION
**Pull request by michaelrkn**

### CODE REVIEW

File: app/models/payment.rb [hunk 1]
```diff
@@ -142,4 +143,9 @@ def check_amount
       throw :abort
     end
   end
+
+  def send_webhook
+    webhook = WebhookPayment.new(self)
+    WebhookJob.perform_later(webhook.endpoint, webhook.payload)
```
Comment (michaelrkn) [inline app/models/payment.rb]: this is hard to follow. how do the webhook.endpoint and webhook.payload values get set?

Comment (michaelrkn) [reply app/models/payment.rb]: also doesn't seem right to call the job here. the webhook should know if it needs to use queuing, not the payment.

Comment (mgoren) [reply app/models/payment.rb]: That makes sense about calling another method that then does the perform_later. Re: webhook.endpoint and webhook.payload, that's a good point... OH, actually moving the actual ActiveJob call into the Webhook class will solve that other issue too. The problem was that perform_later only takes certain basic variable types - not classes.
